lib/active_shipping/carriers/canada_post_pws.rb

```diff
@@ -569,13 +569,13 @@ def log(msg)
 
     def tracking_url(pin)
       case pin.length
-        when 12, 13, 16
-          endpoint + "vis/track/pin/%s/detail" % pin
-        when 15
-          endpoint + "vis/track/dnc/%s/detail" % pin
-        else
-          raise InvalidPinFormatError
-        end
+      when 12, 13, 16
+        endpoint + "vis/track/pin/%s/detail" % pin
+      when 15
+        endpoint + "vis/track/dnc/%s/detail" % pin
+      else
+        raise InvalidPinFormatError
+      end
```

Comment:
Up to you, but while you're here, maybe would be nice to switch to string interpolation:
```ruby
case pin.length
when 12, 13, 16
  "#{endpoint}/vis/track/pin/#{pin}/detail"
when 15
  "#{endpoint}/vis/track/dnc/#{pin}/detail"
else
  raise InvalidPinFormatError
end
```